source/elements/oneMKL/source/domains/spblas/supported-types.rst

```diff
@@ -1,23 +1,20 @@
-.. SPDX-FileCopyrightText: 2019-2020 Intel Corporation
+.. SPDX-FileCopyrightText: 2024 Intel Corporation
 ..
 .. SPDX-License-Identifier: CC-BY-4.0
 
 .. _onemkl_sparse_supported_types:
 
-Supported Types
-===============
-
+Supported template types
+========================
 
 .. container::
 
-
    .. container:: tablenoborder
 
-
-      .. list-table:: 
+      .. list-table::
          :header-rows: 1
 
-         * -  Data Types <fp> 
+         * -  Data Types <fpType> 
```

Comment:
I've always kind of been wary of our use of "fpType" or "fp" to describe the data type since it means floating point type.  At some point there may be requests for int8 or integer valued data in sparse blas routines ...  

What if we use something more appropriately named, like "dataType"  or "valueType" ?

Reply:
Sure, I can rename this to `dataType` and similarly `intType` could be `indexType` to avoid confusion.

Reply:
intType is more tricky since we actually use it to stand for "indexType" (CSR colind, COO rowind/colind) and "offsetType" (CSR rowptr) ..  but sizes like nRow/nCol/nnz are something else, not index or offset per se ...  I think intType is probably fine for now.  What do you think ?

Reply:
I think of the sizes that you mentioned as the upper bound size of an index so it makes sense for indices, offsets and sizes to be of the same type. I think that's pretty common. I have a minor preference for `indexType` if we rename the other parameter to `dataType`, this looks more consistent to me. I can also make that more explicit in this document.

Reply:
ok, but will we call CSR rowptr or CSC colptr an indexType ?   it seems misleading ...

Reply:
ok, I am fine with this,  indexType can represent indices, offsets and sizes ...  I see your argument now.  go ahead :)

Reply:
Done in https://github.com/oneapi-src/oneAPI-spec/pull/522/commits/ce62ad3857a056bb4260a025af948e25f27c3633